### PR TITLE
fix read_at

### PR DESCRIPTION
**Pull request by oltolm**

### CODE REVIEW

**Comment by elad335** [comment]
Was there some sort of regression you are trying to fix? I don't see how it was broken before, seems like here it is broken though.

**Comment by oltolm** [comment]
Yes the parameter `offset` was unused.

File: rpcs3/Emu/Cell/PPUThread.cpp [hunk 1]
```diff
@@ -3533,14 +3533,14 @@ namespace
...
-			const u64 result = file_view::read_at(m_pos, buffer, size);
+			const u64 result = file_view::read_at(m_off + m_pos, buffer, size);
 			m_pos += result;
 			return result;
 		}
 
 		u64 read_at(u64 offset, void* buffer, u64 size) override
 		{
-			return m_file.read_at(m_off + m_pos, buffer, std::min<u64>(size, utils::sub_saturate<u64>(m_max_size, m_pos)));
+			return m_file.read_at(offset, buffer, std::min<u64>(size, utils::sub_saturate<u64>(m_max_size, m_pos)));
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUThread.cpp:3543]: Add m_offs then

File: rpcs3/Emu/Cell/PPUThread.cpp [hunk 2]
```diff
@@ -3533,14 +3533,14 @@ namespace
 
 		u64 read(void* buffer, u64 size) override
 		{
```
Comment (elad335) [inline rpcs3/Emu/Cell/PPUThread.cpp:3535]: drop the change here

File: rpcs3/Emu/Cell/PPUThread.cpp (hunk 1 above)
Comment (oltolm) [reply rpcs3/Emu/Cell/PPUThread.cpp]: done

File: rpcs3/Emu/Cell/PPUThread.cpp (hunk 2 above)
Comment (oltolm) [reply rpcs3/Emu/Cell/PPUThread.cpp]: done

**Review by elad335 — APPROVED** [review]

**Pull request merged**